### PR TITLE
Chen_Gutman - fullJustify

### PR DESCRIPTION
**Pull request by chengutman**
Given an array of words and a width maxWidth, format the text such that each line has exactly maxWidth characters and is fully (left and right) justified

### CODE REVIEW

**Comment by omerbensimon** [comment]
Great code!
Comments Where Needed

File: hw1_interview_questions/chen_gutman [hunk 1]
```diff
@@ -0,0 +1,52 @@
+function fullJustify(words: string[], maxWidth: number): string[] {
```
Comment (barakdaniel) [inline hw1_interview_questions/chen_gutman:1]: An explanation about the function, its input and output would help.

File: hw1_interview_questions/chen_gutman [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            remainingSpaces--;
+        }
+        
+        return parts.join('');
+    }
+    
+    function fitsOnLine(word, l) {
+        return (l + word).length <= maxWidth;
+    }
+};
```
Comment (barakdaniel) [inline hw1_interview_questions/chen_gutman:52]: Great use of ts/js built in behavior.

**Comment by yoav-zibin** [comment]
No tests

**Pull request merged**